.bootstraprc

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
We should define and check only 3 or 4. Other should be in user land. It's gonna work as long as paths to js scss won't be changed. for major version of bootstrap.